### ISSUE TITLE
CSS conflicts with Foundation

### ISSUE THREAD

**Issue by bikedorkjon** [zurb/flickrbomb#1]
Doesn't work out of the box with Foundation.

.flickrbombWrapper doesn't inherit height from img height.
img.flickrbomb element.style has top: -79px;

**Comment by mkelly12** [comment]
That's correct. FlickrBomb was not built to work with Foundation. We don't currently have any plans to rewrite the CSS to work with Foundation, but we'll keep this issue open since it's defiantly a valid problem.
